packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -88,9 +92,7 @@ export const getCheckboxStyles = stylesFactory((theme: GrafanaTheme2) => {
        * for angular components styling
        * */
       &:checked + span {
-        background: blue;
         background: ${theme.colors.primary.main};
-        border: none;
```

Comment:
Removing this changes the visual of the ticked checkbox (you can see it in your screenshot)

Reply:
I can't notice it (silly eyes), but I did mean to undo this change anyway because it's unrelated 😌